Hu-Fi/hufi
# add `localhost` environment to the services

## Description


Adding `localhost` environment to the services.

## Summary of changes


- Campaign Launcher UI
  - Removed `useChainId` which is not returning back the current chain ID connected from the wallet. Using `useAccount` instead.
  - Added USDT/HUSD contract address for localhost.
  - Added localhost to supported chain ids, in case of development server.
- Campaign Launcher Server / Recording Oracle / Reputation Oracle
  - Updated `.env.example` for localhost environment.
  - Added RPC_URL_LOCALHOST for localhost RPC connection.
- Updated README.

## How to test the changes


- Make sure we can run the services locally with steps described in the README.
- Make sure localhost is not displayed from vercel deployment.

## Related issues
[Keywords for linking issues](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests)


Closes #44
